templates/partials/department_card.html

```diff
@@ -12,17 +12,21 @@ <h4 class="card-subtitle">Top-spending departments in {{ entity }}</h4>
         <tr>
           <th scope="col">Department</th>
           <th scope="col" class="text-right">Number of employees</th>
-          <th scope="col" class="text-right">Median salary</th>
+          <th scope="col" class="text-right">Median total pay</th>
```

Comment:
This is looking super awesome! How do you feel about explicitly setting the table column width, so these figures are evenly spaced? I think it's [pretty easy with Bootstrap](https://www.codeply.com/p/3ZUx3BynZD0)!

Reply:
yeah, that would be nice! I did mess around with the columns a bit but didn't get very far on my own. Could you link to that article again, @hancush ? I got a 404 page

Reply:
Here's the plain link: https://www.codeply.com/p/3ZUx3BynZD!